src/test/scala/io/viash/package_/PackageTest.scala

```diff
@@ -1,12 +1,12 @@
-package io.viash.project
+package io.viash.package_
```

Comment:
Should this not be moved to packageConfig` instead of `package_`?

Reply:
Thanks for spotting it!